tuning-profiles/custom-hiera-ex-large-128G.yaml

```diff
@@ -0,0 +1,60 @@
+# Tuning suitable for a Satellite with 128GG of RAM
+#
+# Instructions for use:
+# 1) Copy this file to /etc/foreman-installer/custom-hiera.yaml
+# 2) run 'satellite-installer' with no arguments to apply these settings
+
+apache::mod::passenger::passenger_max_pool_size: 120
+apache::mod::passenger::passenger_max_request_queue_size: 1000
+apache::mod::passenger::passenger_max_requests: 1000
+
+apache::mod::prefork::serverlimit: 1024
+apache::mod::prefork::maxclients: 1024
+apache::mod::prefork::maxrequestsperchild: 4000
+
+qpid::open_file_limit: 65536
+qpid::router::open_file_limit: 150100
+
+candlepin::java_opts: "-Xms1024m -Xmx8192m"
+
+postgresql::server::config_entries:
+  max_connections: 1000
+  shared_buffers: 16GB
+  work_mem: 8MB
+  checkpoint_segments: 32
+  checkpoint_completion_target: 0.9
+  effective_cache_size: 32GB
+  autovacuum_vacuum_cost_limit: 2000
+  log_min_duration_statement: 500
+
+
+# Additional considerations:
+#
+# For really high # of concurrency and the need to go to 256 PassengerMaxPoolSize or above
+# you need to tune the kernel:
+#
+# 1) persist this configuration so it survives reboots:
+#
+# echo "kernel.sem= 250 256000 32 16384" > /etc/sysctl.d/01-satellite-tune.conf
```

Comment:
Created an RFE to get support for this in the installer 

https://bugzilla.redhat.com/show_bug.cgi?id=1715551

Reply:
:+1: